### ISSUE TITLE
It does not work at all on some websites

### ISSUE THREAD

**Issue by Korios** [Zemke/instant-smart-quotes#26]
I tried it on Medium, Twitter, Youtube, Facebook and Vocal (vocal.media). It works only on Twitter and Youtube, it half-works on FB and Medium (it works on new posts but not comments…) and it does not work _at all_ with Vocal’s text editor. 

Above all I needed it for Vocal, so I do not have to write everything at Writer first or waste time with the virtual keyboard (I use Linux) and/or multiple keyboard shortcuts designed to be impossible to use (unless you are a keyboard virtuoso), but it does not even register with Vocal… I verified that it is active during a Vocal new post session, but Vocal’s quotes do not react all all.. 
Oh, last but not least, I just found out it works with Github.

**Comment by Zemke** [comment]
Yeah, some websites have so much own complexity running on their text fields that it’s simply incompatible with this add-on unfortunately. Haven’t yet figured out how to work around it.
